app/assets/javascripts/campaign.js

```diff
@@ -34,4 +34,46 @@ $(function() {
 			alert(response.error_description);
 		}
 	});
-});
\ No newline at end of file
+});
+
+$(function()
```

Comment:
You can just add the new calls in the existing $(function(){ block without having to make new ones.